FusionIIIT/applications/placement_cell/migrations/0002_auto_20230407_1155.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.5 on 2023-04-07 11:55
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('academic_information', '0002_student_hall_id'),
+        ('placement_cell', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='placementrecord',
+            name='branch',
+            field=models.CharField(default='', max_length=100),
+        ),
```

Comment:
@vanshksharma the migrations for branch are already present on the test you can see them [here](https://github.com/FusionIIIT/Fusion/blob/test/FusionIIIT/applications/placement_cell/migrations/0003_placementrecord_branch.py). Remove these lines here.